pkg/models/resources/pods.go

```diff
@@ -221,6 +221,14 @@ func (*podSearcher) fuzzy(fuzzy map[string]string, item *v1.Pod) bool {
 
 func (*podSearcher) compare(a, b *v1.Pod, orderBy string) bool {
 	switch orderBy {
+	case StartTime:
+		if a.Status.StartTime == nil {
+			return true
+		}
+		if b.Status.StartTime == nil {
+			return false
+		}
+		return a.Status.StartTime.Before(b.Status.StartTime)
```

Comment:
if a is older than b. `true`
if a is not started . `true`
if a is started and b is not started `false` / / If b is not started, it should be false？

Reply:
fixed